backend/src/test/java/de/jankahle/capstone/TestFactory.java

```diff
@@ -18,6 +18,7 @@ public static Recipe createPotatoRecipe() {
 
     public static RecipeDto createPotatoDto() {
         return RecipeDto.builder()
+                .id("")
```

Comment:
You use the builder inside your `RecipeDto`. Why set `id` to `""` manually here? Am I missing something?

Reply:
As String is an object-type the default value is `null`, therefore if there is any kind of method that tries to work with id it will throw nullref exceptions if there is not at least an empty String assigned.

Reply:
I merely noted it because you didn't replace the zero with an empty string in the running code outside of the tests. Or did i miss something?